src/site/content/en/handbook/content-types/example-landing-page/index.njk

```diff
@@ -0,0 +1,147 @@
+---
+layout: landing-page
+title: A sample landing page
+description: This page is used for visual testing of the landing-page layout and also showcases it's capabilities
+headlineAction:
+  url: /handbook/content-types/example-landing-page/
+  text: Test
+paged:
+  elements:
+    - data:
+        title: >-
+          A post without a thumbnail
+        subhead: >-
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          In et eros volutpat, commodo neque quis, egestas tortor
+        url: /handbook/content-types/example-landing-page/
+        thumbnail:
+        alt: Lorem Ipsum
+        authors:
+          - razvancaliman
```

Comment:
It might be better to use an example author here, following https://github.com/GoogleChrome/developer.chrome.com/pull/4791 - that's also true for the other example posts; though I know it's not ideal to test multiple authors...